deploy/charts/cert-manager/values.secure-defaults.yaml

```diff
@@ -0,0 +1,110 @@
+# Helm chart values which make cert-manager comply with CIS, BSI and NSA
+# security benchmarks.
+# Read https://cert-manager.io/docs/installation/best-practice for more
+# information.
+automountServiceAccountToken: false
+serviceAccount:
+  automountServiceAccountToken: false
+volumes:
+- name: serviceaccount-token
+  projected:
+    defaultMode: 420
```

Comment:
It's recommended to use 292 here as an octal value representing unix 0400 permissions

Reply:
Done. I used the octal value `0444` for readability.
`0400` is not sufficient (I think) because the cert-manager processes run as non-privileged user IDs inside the containers.

Reply:
```
richard@LAPTOP-HJEQ9V9G:~/projects/cert-manager/cert-manager$ kubectl -n cert-manager get pod cert-manager-6844cc9bbb-6tf8l -o yaml | grep -C 10 292
    key: node.kubernetes.io/not-ready
    operator: Exists
    tolerationSeconds: 300
  - effect: NoExecute
    key: node.kubernetes.io/unreachable
    operator: Exists
    tolerationSeconds: 300
  volumes:
  - name: serviceaccount-token
    projected:
      defaultMode: 292
      sources:
      - serviceAccountToken:
          expirationSeconds: 3607
          path: token
      - configMap:
          items:
          - key: ca.crt
            path: ca.crt
          name: kube-root-ca.crt
      - downwardAPI:
```